src/System.IO.FileSystem.AccessControl/tests/Configurations.props

```diff
@@ -2,7 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard-Windows_NT;
+      netfx-Windows_NT;
```

Comment:
Great! one more ask though, do you mind running `msbuild /t:UpdateVSConfigurations build.proj` from the root of corefx in order to also update in this PR the .csproj for tests so that we don't break VS build?